src/test/java/com/epam/tc/hw7/test/JdiBaseTest.java

```diff
@@ -0,0 +1,37 @@
+package com.epam.tc.hw7.test;
+
+import static com.epam.jdi.light.elements.init.PageFactory.initSite;
+import static com.epam.tc.hw7.pages.JdiSite.login;
+import static com.epam.tc.hw7.pages.JdiSite.logout;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.epam.tc.hw7.data.PropertyInitialization;
+import com.epam.tc.hw7.data.UserCreator;
+import com.epam.tc.hw7.model.User;
+import com.epam.tc.hw7.pages.HomePage;
+import com.epam.tc.hw7.pages.JdiSite;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.BeforeSuite;
+
+public class JdiBaseTest {
```

Comment:
Why you create this class ? Can you just use Before annotation in JdiSiteTest ?

Reply:
I organized it in order to expand my tests in the future and move common steps there. But now it would better to use before and after annotations. I`ve fixed it.